examples/atlas/create-cluster.md

```diff
@@ -0,0 +1,15 @@
+# Create a Cluster
+
+To create and get the connection string for a cluster you can do:
+
+```bash
+mongocli atlas clusters create MyCluster \
+  --region EASTERN_US \
+  --members 3 \
+  --tier M30 \
+  --provider GCP \
+  --mdbVersion 4.4 \
+  --diskSizeGB 30 && \
+mongocli atlas clusters watch MyCluster && \
+mongocli atlas clusters describe MyCluster -o go-template="{{.SrvAddress}}"
```

Comment:
[q] what about inserting the definition of each flag, something like this
```suggestion
mongocli atlas clusters create MyCluster \
  --region EASTERN_US \ # Physical location of your MongoDB cluster.
  --tier M30 \ # Tier for each data-bearing server in the cluster. 
  --provider GCP \ # Name of your cloud service provider.
  --mdbVersion 4.4 \ # MongoDB version of the cluster to deploy. 
  --diskSizeGB 30 && \ # Capacity, in gigabytes, of the host’s root volume.
mongocli atlas clusters watch MyCluster && \
mongocli atlas clusters describe MyCluster -o go-template="{{.SrvAddress}}"
```

Reply:
this was a nice idea but I just tried to copy/paste the new command and it no longer works so I won't add this